app/views/necesidad/necesidades.html

```diff
@@ -1,17 +1,23 @@
+<style>
+  #listaNec{
+    border: 0;
+    width: 100%;
+    height: 60vh;
+  }
+  </style>
```

Comment:
```suggestion
#listaNec{
  border: 0;
  width: 100%;
  height: 60vh;
}
</style>
```

De ser posible, mejor mover este estilo a alguno de los archivos en `app/styles`, con el fin de que los HTML por mucho tengan estilos dinámicos, que lo estáticos se traigan de otro lado. Se podría crear un archivo en esa carpeta llamado `necesidades.css`

Adicionalmente, en vez de `id` mejor usar `class` (<-- Esto ultimo no es estrictamente necesario, es una recomendación)